task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -47,11 +50,13 @@ spec:
           fi
         done
         if [ -z "$FAILEDIMAGES" ]; then
-          HACBS_TEST_OUTPUT="$(make_result_json -r SUCCESS -s 1)"
+          note="Task $(context.task.name) succeeds, please check tekton task result for the details"
+          HACBS_TEST_OUTPUT="$(make_result_json -r SUCCESS -s 1 -t $note)"
           echo "${HACBS_TEST_OUTPUT}" | tee "$(results.HACBS_TEST_OUTPUT.path)"
         else
           echo "These images failed inspection: $FAILEDIMAGES"
-          HACBS_TEST_OUTPUT="$(make_result_json -r FAILURE -f 1)"
+          note="Task $(context.task.name) meets error: Fails to inspect images, please check the tekton task log"
```

Comment:
SR: note="Task $(context.task.name) Failed: Fails to inspect images. Check the Tekton task log."

what failed to inspect the images?

Reply:
The task completed without any error (exit 0), but the result of the result `HACBS_TEST_OUTPUT` is `FAILURE`, it means the image doesn't pass the test. So I feel we can't say `Task $(context.task.name) failed`, maybe we can say `Test $(context.task.name) failed`?